.github/workflows/integration.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       with:
         path: faucet
         repository: bosagora/faucet
-        submodules: true
+        submodules: recursive
 
     - name: Update Faucet submodule to point to our clone
```

Comment:
I think you need to run `git submodule update --init` before the "For debug output", otherwise the problem will still show.

Reply:
https://github.com/bosagora/agora/runs/4520765196?check_suite_focus=true it builds fine

Reply:
Now try updating a submodules with a breaking change

Reply:
`git submodule update --init`

Added that in as well

Reply:
> Now try updating a submodules with a breaking change

🤣 https://github.com/bosagora/agora/runs/4520736505?check_suite_focus=true